libs/blocks/editorial-card/editorial-card.js

```diff
@@ -80,38 +81,50 @@ function handleClickableCard(el) {
 
 const init = async (el) => {
   el.classList.add('con-block');
-  if (el.className.includes('open')) {
-    el.classList.add('no-border', 'l-rounded-corners-image', 'static-links-copy');
-  }
-  if (el.className.includes('rounded-corners')) {
-    loadStyle(`${base}/styles/rounded-corners.css`);
-  }
+  const hasOpenClass = el.className.includes('open');
+  if (hasOpenClass) el.classList.add('no-border', 'l-rounded-corners-image', 'static-links-copy');
+  if (el.className.includes('rounded-corners')) loadStyle(`${base}/styles/rounded-corners.css`);
```

Comment:
I was doing a performance review recently and notices that the `rounded-corners.css` file tends to get loaded post LCP files like `brick.css` and `text.css`, yet it is a relatively small amount of CSS. Is there a reason this CSS cannot be included elsewhere and load in with other files? Even `styles.css`?

Reply:
Im open to making this a more global style etc. and loading it accordingly. Mostly im just following suit here w/ other blocks that use this as an opt in style, if it's authored. I think if we want to move these styles all together it should be a separate task since it impacts many blocks.